core-services/release-controller/_releases/release-okd-4.y-stable.json

```diff
@@ -2,7 +2,6 @@
   "name":"4-stable",
   "as": "Stable",
   "message": "All amd64 OKD versions are tracked here",
-  "overrideCLIImage": "image-registry.openshift-image-registry.svc:5000/ocp/4.6:cli",
```

Comment:
Is there any specific reason that you removed this?
This is a release-controller specific setting that should remain in place.